gems/smithy/spec/fixtures/shapes/model.smithy

```diff
@@ -0,0 +1,115 @@
+$version: "2"
+
+namespace smithy.ruby.tests
+
+@paginated(inputToken: "nextToken", outputToken: "nextToken", pageSize: "pageSize")
+service ShapeService {
+    version: "2018-10-31"
+    resources: [
+        SomeResource
```

Comment:
Probably not necessary to have a resource anyway. A single operation is fine.

Reply:
I think we should have a resource here to ensure that the operation defined within the resource is created

Reply:
We already test that operations are provided in the operation index if they are defined in resources.

Reply:
That's fair, I will remove.